sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_pii_entities.py

```diff
@@ -87,11 +87,69 @@ def test_all_successful_passing_text_document_input(self, client):
 
     @GlobalTextAnalyticsAccountPreparer()
     @TextAnalyticsClientPreparer()
-    def test_length_with_emoji(self, client):
+    def test_emoji(self, client):
         result = client.recognize_pii_entities(["👩 SSN: 859-98-0987"])
         self.assertEqual(result[0].entities[0].offset, 7)
         self.assertEqual(result[0].entities[0].length, 11)
 
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
```

Comment:
this tests are not specifically for PII. Looks like Python has files per group of tests, so consider moving this to a specific file and test it for different endpoints too

Reply:
I'll move this into another test file for offset and length tests